db/attachment.go

```diff
@@ -246,6 +251,9 @@ func (db *Database) setAttachment(key string, value []byte) error {
 func (db *Database) setAttachments(attachments AttachmentData) error {
 	for key, data := range attachments {
 		attachmentSize := int64(len(data))
+		if attachmentSize > int64(gomemcached.MaxBodyLen) {
```

Comment:
Can we define our own constant for doc size? `gomemcached` is virtually unused (directly) by SG and I'm not sure it makes sense for that library to determine what we do or don't send via GoCB.